addons/im_livechat/models/im_livechat_channel_member_history.py

```diff
@@ -193,16 +193,14 @@ def action_open_discuss_channel_view(self, domain=()):
         action["views"] = [view for view in action["views"] if view[1] in ("list", "form")]
         return action
 
-    def _to_store_defaults(self, target):
-        return [
-            "channel_id",
-            Store.One("guest_id", ["name"], predicate=lambda r: not r.partner_id),
-            "livechat_member_type",
-            # sudo: res.partner - reading partner related to an accessible channel member history is considered acceptable
-            Store.One(
-                "partner_id",
-                self.env["res.partner"]._get_store_livechat_username_fields(),
-                predicate=lambda r: not r.guest_id,
-                sudo=True,
-            ),
-        ]
+    def _store_member_history_fields(self, res: Store.FieldList):
+        res.attr("channel_id")
+        res.one("guest_id", ["name"], predicate=lambda r: not r.partner_id)
+        res.attr("livechat_member_type")
```

Comment:
```suggestion
    def _store_member_history_fields(self, res: Store.FieldList):
        res.one("guest_id", ["name"], predicate=lambda r: not r.partner_id)
        res.extend(["channel_id", "livechat_member_type"])
```
:pick:

Reply:
Ok to change to `extend` when fields are following each other, but when they are not, it might change order, which might requires more changes in tests (and could also potentially change prefetch, query count, etc).

I think it's ok to do because "attr" fields are all fetched together in Store, but I would prefer to do it separately.